app/src/main/kotlin/com/wire/android/notification/CallNotificationManager.kt

```diff
@@ -57,13 +56,15 @@ class CallNotificationManager @Inject constructor(private val context: Context)
     private fun createIncomingCallsNotificationChannel() {
         val audioAttributes = AudioAttributes.Builder()
             .setContentType(AudioAttributes.CONTENT_TYPE_MUSIC)
-            .setUsage(AudioAttributes.USAGE_MEDIA)
+            .setUsage(AudioAttributes.USAGE_NOTIFICATION_RINGTONE)
```

Comment:
@alexandreferris It seems this did the trick for me 🤯 Now the vibration is stopping 👌